maestro_ctrl

```diff
@@ -190,6 +192,8 @@ class Cluster(object):
                     'reconnect' : reconnect,
                     'updaters'  : upd_spec
                 }
+                if subscribe:
```

Comment:
@nick-enoent Bug: This check (maestro_ctrl:195) assumes that subscribe was defined in the outer scope prod[]. subscribe needs to be set = None  at about line 176 if it is not defined in prod[].
maestro_ctrl fails with a stack blob if not subscribe in prod[].

A better coding of this would not use shadow variable naming (i.e. use nprod = {...} in the for loop body and append nprod to producers.group).